vivarium/experiments/hierarchy_composite_division.py

```diff
@@ -71,7 +71,7 @@ def test_hierarchy_update() -> None:
                 'Protein': {'X': 50 * units.mg / units.mL}}}}
 
     # make a txtl composite, embedded under an 'agents' store
-    txtl_composer = AgentDivision({})
+    txtl_composer = AgentDivision({'growth': {'default_growth_noise': 0}})
```

Comment:
This test randomly failed on me once, so I'm hoping this change makes it more reproducible.